healthcheck/webconsoleplugin/bnd.bnd

```diff
@@ -1,10 +1,6 @@
-Bundle-Category: healthcheck
-
-Bundle-Description: ${project.description}
-
-Bundle-DocURL: https://felix.apache.org
-
-Bundle-License: Apache License, Version 2.0
-
-Bundle-Vendor: The Apache Software Foundation
-
+Bundle-Category      : healthcheck
+Bundle-Description   : ${project.description}
+Bundle-DocURL        : https://felix.apache.org
+Bundle-License       : Apache License, Version 2.0
+Bundle-Vendor        : The Apache Software Foundation
+-conditionalpackage  : org.apache.commons.lang3.*
```

Comment:
For an embedded use case I assume the webconsole is not relevant - I would not make changes here if not needed.

Reply:
Actually an embedded use case comprises both production-ready and developer-friendly gateways. I agree with you that such a web console bundle is not relevant for a production ready gateway but it is definitely required for developers in the development phase.

Reply:
So for server applications we do use the webconsole in production, I just thought that for `small IoT devices` you wouldn't use it at all (and maybe rather the gogo shell). But if you use it that's fine obviously, it just makes it one bundle more in your setup and the use case for embedding even weaker.